scripts/mustgather/kabanero-mustgather.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Run this script to collect debug information
+
+set -Euox pipefail
+
+COMPONENT="kabanero.io"
+BIN=oc
+LOGS_DIR=kabanero-debug
+
+# Describe and Get all api resources of component across cluster
+
+APIRESOURCES=$(${BIN} get crds -o jsonpath="{.items[*].metadata.name}" | tr ' ' '\n' | grep ${COMPONENT})
```

Comment:
I think that we're going to want to also get the resources that get deployed by the collections.  It's sort of impossible right now to read the collection and see what should be there, but I think tekton pipelines and tasks, and roles / cluster roles would be good.